server/views/applications/pages/accommodation-need/contact-details/practitioner-email.njk

```diff
@@ -0,0 +1,22 @@
+{% extends "../../layout.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block questions %}
+
+    <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ task.title }}</span>
+        {{ page.title }}
+    </h1>
+
+    {{ formPageInput({
+        fieldName: "email",
+        label: {
+            text: "Email address"
+        }
+    }, fetchContext()) }}
```

Comment:
Could we start labeling input fields using the page heading instead of "Email address"? Without the page heading theres no context what the user is meant to record. In this case, "Email address"  adds very little value and it would better to use the page heading "Update probation practitioner email address".

Reply:
I've added a comment on the ticket to ask Ray for his opinion on this.